tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -945,16 +727,70 @@ def writable_dir(path: str):
             raise Error('The directory "{}" is not writable'.format(cur_path))
 
 
+def add_args_by_description(args_group, params_description):
+    signature = inspect.signature(openvino.tools.mo.convert_model)
+    filepath_args = get_params_with_paths_list()
+
+    special_actions = {'transformations_config': CanonicalizeTransformationPathCheckExistenceAction,
+                       'input_model': CanonicalizePathCheckExistenceAction,
+                       'extensions': CanonicalizeExtensionsPathCheckExistenceAction,
+                       'counts': CanonicalizePathCheckExistenceIfNeededAction,
+                       'version': 'version'}
+    special_types = {'input_model': readable_file_or_dir, 'scale': float, 'extensions': readable_dirs_or_files_or_empty, 'batch': check_positive}
+    param_aliases = {'input_model': {'-w', '-m'}, 'scale': {'-s'}, 'batch': {'-b'}, 'input_proto': {'-d'}}
```

Comment:
please create special dictionary to map help `parameter` for `cli_parser`

Reply:
Done.